coralproject/talk
# [CORL-2300] Store and re-use detected words for mod card highlighting

## What does this PR do?

Persists the detected words to the wordList metadata so that we can use it to highlight the moderate cards instead of re-doing the regex detection we already did.

## What changes to the GraphQL/Database Schema does this PR introduce?

Added banned and suspect words to the wordList metadata on comments.

## How do I test this PR?

- Add some suspect and banned words to your word lists
- Create comments with these words
- Check mod queue and see the words highlighted on these comments
 
## How do we deploy this PR?

Same as any feature, just deploy with the release from develop to main.
